src/files/dropdown/Dropdown.js

```diff
@@ -1,10 +1,22 @@
 import React, { forwardRef } from 'react'
 import { Dropdown as Drop, DropdownMenu as Menu } from '@tableflip/react-dropdown'
+import CopyIcon from '../../icons/CopyIcon'
 
-export const Option = ({ children, onClick, className = '', ...props }) => (
-  <button role="menuitem" className={`bg-animate hover-bg-near-white pa2 pointer flex items-center ${className}`} onClick={onClick} {...props}>
-    {children}
-  </button>
+export const Option = ({ children, onClick, className = '', isCliTutorModeEnabled, onCliTutorMode, ...props }) => (
+  isCliTutorModeEnabled
+    ? <div className='flex items-center justify-between'>
+      <button role='menuitem' className={`bg-animate hover-bg-near-white pa2 pointer flex items-center flex-grow-1 ${className}`} onClick={onClick} {...props}>
+        {children}
+      </button>
+      <button {...props} className={`bg-animate hover-bg-near-white pa2 pointer flex items-center  ${className}`}>
+        <CopyIcon {...props} onClick={onCliTutorMode} className='dib fill-current-color ph2 glow o-80 pointer'
+          style={{ height: '28px', transform: 'scale(1.3)', color: 'dodgerblue' }}
```

Comment:
Try to avoid the transform (same as commented before)

Reply:
added a css class for this with `webkitTransform` but it would also work if we simple increased the height by 1.3X. Lemme know which way we'd like to move forward with :)